src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -148,21 +152,33 @@ struct ConstantLifetimeChargeTrappingModel{T <: SSDFloat, G <: Union{<:AbstractG
     inactive_layer_geometry::G
 end
 
+is_inactive_layer(point_types::Array{UInt8, 3}) = any(val -> (val & inactive_layer_bit) != 0, point_types)
```

Comment:
```suggestion
is_inactive_layer(point_types::Array{PointType, 3}) = any(val -> (val & inactive_layer_bit) != 0, point_types)
```

Also: shouldn't this be called something like `has_inactive_layer` rather than `is_inactive_layer`?

Reply:
Done